### DISCUSSION TITLE
Recap for October 16th, 2024 - Semantic Kernel Office Hours (US/EMEA)

### DISCUSSION THREAD

**Discussion by sophialagerkranspandey** [microsoft/semantic-kernel#9312] [Office Hours Recaps]
Semantic Kernel Office Hours for US/EMEA - October 16th, 2024 recording: https://www.youtube.com/watch?v=uoeluhfCxbw&list=PLmsFUfdnGr3xAtM3ofjpEkIWnuSA2d-i-&index=37

If you weren't able to attend office hours but still have questions, please post them here: https://github.com/microsoft/semantic-kernel/discussions

![image](https://github.com/user-attachments/assets/66ba54a5-1a3d-418a-9e3c-005d73c3f44c)

**Comment by JP-droidic** [comment]
Thanks, @sophialagerkranspandey, for sharing notes. Unfortunately, I wasn't able to attend the call, but I have a follow-up question about the PDF to RAG demo shown on the call.

How to deal with embedded images in a PDF document? Any suggestions? What I noticed is when I upload a PDF document directly to chatGPT interfaces it is able to answer/return knowledge that is part of the image, and it is super fast. Any ideas how to mimic this?

**Reply by sophialagerkranspandey** [reply]
When you're parsing through the PDF, you can send the image to an image to text model to get a description and generate an embedding and add it in your vector database to store and do your search against that.
